requirements.dev.txt

```diff
@@ -17,7 +17,7 @@ pytest==4.3.1
 selenium==3.141.0
 snowballstemmer==1.2.1    # via sphinx
 sphinx-rtd-theme==0.4.3
-sphinx==2.0.0
+sphinx==2.0.0;python_version >= '3'
```

Comment:
don't we need sphinx to build the documentation on py2 systems?

Reply:
@pbugni 
looks like they're pulling the plug already. we can wait to update sphinx too

>Incompatible changes
>--------------------
>
>* Drop python 2.7 and 3.4 support

Reply:
as long as the build still functions at https://readthedocs.org/ I see no need to keep it.  we're clearly migrating away from py2, and this is likely going to be a new pattern.  more motivation :)

Reply:
I just checked RTD and after fixing some OAuth settings I was able to test with a [python3 interpreter, looked good!](https://readthedocs.org/projects/truenth-shared-services/builds/8867757/)